nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-processors/src/main/resources/docs/org.apache.nifi.processors.script.ExecuteScript/additionalDetails.html

```diff
@@ -0,0 +1,698 @@
+    </tr>
+    <tr>
+        <td><b>context</b></td>
+        <td>This is a reference to the ProcessContext for the processor. It can be used to retrieve processor properties, relationships, Controller Services, and the StateManager.</td>
+        <td><a href="https://www.javadoc.io/doc/org.apache.nifi/nifi-api/latest/org/apache/nifi/processor/ProcessContext.html">ProcessContext</a></td>
+    </tr>
+    <tr>
+        <td><b>log</b></td>
+        <td>This is a reference to the ComponentLog for the processor. Use it to log messages to NiFi, such as log.info('Hello world!')</td>
+        <td><a href="https://www.javadoc.io/doc/org.apache.nifi/nifi-api/latest/org/apache/nifi/logging/ComponentLog.html">ComponentLog</a></td>
+    </tr>
+    <tr>
+        <td><b>REL_SUCCESS</b></td>
+        <td>This is a reference to the "success" relationship defined for the processor. It could also be inherited by referencing the static member of the parent class (ExecuteScript), but some engines such as Lua do not allow for referencing static members, so this is a convenience variable. It also saves having to use the fully-qualified name for the relationship.</td>
+        <td><a href="https://www.javadoc.io/doc/org.apache.nifi/nifi-api/latest/org/apache/nifi/processor/Relationship.html">Relationship</a></td>
+    </tr>
+    <tr>
+        <td><b>REL_FAILURE</b></td>
+        <td>This is a reference to the "failure" relationship defined for the processor. As with REL_SUCCESS, it could also be inherited by referencing the static member of the parent class (ExecuteScript), but some engines such as Lua do not allow for referencing static members, so this is a convenience variable. It also saves having to use the fully-qualified name for the relationship.</td>
+        <td><a href="https://www.javadoc.io/doc/org.apache.nifi/nifi-api/latest/org/apache/nifi/processor/Relationship.html">Relationship</a></td>
+    </tr>
+    <tr>
+        <td><i>Dynamic Properties</i></td>
+        <td>Any dynamic (user-defined) properties defined in ExecuteScript are passed to the script engine as variables set to the PropertyValue object corresponding to the dynamic property. This allows you to get the String value of the property, but also to evaluate the property with respect to NiFi Expression Language, cast the value as an appropriate data type (such as Boolean, e.g.), etc. Because the dynamic property name becomes the variable name for the script, you must be aware of the variable naming properties for the chosen script engine. For example, Groovy does not allow periods (.) in variable names, so an error will occur if "my.property" was a dynamic property name.
+            Interaction with these variables is done via the NiFi Java API, the 'Dynamic Properties' section below will discuss the relevant API calls as they are introduced. </td>
```

Comment:
Suggest breaking this into two sentences:

...NiFi Java API.  The 'Dynamic Properties' section...